apps/src/templates/sectionProgress/progressTables/ProgressTableContainer.jsx

```diff
@@ -251,5 +253,6 @@ export default connect(state => ({
     state.sectionProgress.studentLastUpdateByScript[
       state.scriptSelection.scriptId
     ],
-  localeCode: state.locales.localeCode
+  localeCode: state.locales.localeCode,
+  showSectionProgressDetails: state.sectionProgress.showSectionProgressDetails
```

Comment:
Where is this used?

Reply:
`ProgressTableStudentName` line 103. that component used to be getting it from redux direction, but since it's part of the scrolling table i wanted to make it lighter-weight.

Reply:
It might be worth adding a comment about that where it's being passed in so that someone doesn't come along and revert this change accidentally.